test/util/test_lock.py

```diff
@@ -0,0 +1,109 @@
+"""Test lock utility functions."""
+from unittest.mock import patch
+
+import pytest
+
+from zwave_js_server.const import ATTR_CODE_SLOT, ATTR_IN_USE, ATTR_NAME, ATTR_USERCODE
+from zwave_js_server.exceptions import NotFoundError
+from zwave_js_server.util.lock import (
+    get_code_slots,
+    get_usercode,
+    get_usercodes,
+    set_usercode,
+)
+
+CODE_SLOT_NAME = "User Code ({})"
+
+
+def test_get_code_slots(lock_schlage_be469):
+    """Test get_code_slots utility function."""
+    node = lock_schlage_be469
+    code_slots = get_code_slots(node)
+    for x in range(0, 30):
+        code_slot = x + 1
+        in_use = False
+        if x < 3:
```

Comment:
Define a constant list of dicts instead with `code_slot`, `in_use` and `user_code` items for all code slots.